docs/source/gbd2019_models/causes/ischemic_stroke/index.rst

```diff
@@ -159,38 +156,23 @@ The following table describes any restrictions in GBD 2019 on the effects of thi
 Vivarium Modeling Strategy
 --------------------------
 
-.. todo::
-
-  Add an overview of the Vivarium modeling section.
 
 Scope
 +++++
 
-.. todo::
+This cause model is designed to simulate the occurrence of first and recurrent acute ischemic stroke. It is not designed to simulate recurrent events where the second event is a different type of stroke. When incorporating risk factors, BMI, SBP, LDL cholesterol, smoking, FPG, physical inactivity, and total alcohol inactivity should affect transition rates 1 and 3 through the GBD measure of incidence for each stroke cause. 
```

Comment:
These risk/outcome relationships will get detailed in a Risk Effect Model Document, and when you make that doc, you can link from here to there.